DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Maybe its useful here to have a brief sentence explaining what format for the table is expected by the TAP service, or what formats are generally allowed? It looks like you are reading it in using pandas and then converting it to an astropy table, but does it have to be an astropy table? Or can it just be arrays of values? Wondering if the pandas step us necessary or if one could just use astropy ascii.read in 1 line.</p><p> </p><p>Generally it might be nice to start Section 3 with an overview sentence of the purpose or advantage of this feature. At the moment I focus on ADQL part of the title and I didn't exactly understand what part of this is ADQL (just thinking back to your slack question where all ADQL inner joins are just joins?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Definitely a pandas table is not working. I made this part simpler, by not converting the result table to pandas dataframe in the first place. Astropy tables seem to the expected table format, but if a table includes unicode column, it is not acceptable. I am trying to understand some details, but it's something deeper in the database. Need to ask some help from the dm_rsp team. </p><p> </p><p>I'm not sure if I understand your comment about ADQL - I copied this title from your portal tutorial. This part is to join a user-uploaded table to a DP0.3 table using ssObjectId. Do you think the title needs to be something else?</p><p> </p><p> </p><p> </p>

Reply:
<p>Added a sentence to improve the clarity</p><p> </p>